apps/examples/performance/tls_seclink/Kconfig

```diff
@@ -0,0 +1,46 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config EXAMPLES_TLS_SECLINK
+	bool "TLS Loopback Test with SECLINK"
+	default n
+	depends on NET_SECURITY_TLS
+
+if EXAMPLES_TLS_SECLINK
+
+config EXAMPLES_TLS_SECLINK_PROGNAME
+	string "Program name"
+	default "tls_seclink"
+	depends on BUILD_KERNEL
```

Comment:
you mean protected build or something? BUILD_KERNEL is not used anywhere now. please check.

Reply:
@jeongarmy Ok, I removed it.